src/System.Private.CoreLib/shared/System/RuntimeType.cs

```diff
@@ -187,8 +187,6 @@ protected override TypeCode GetTypeCodeImpl()
                     typeCode = TypeCode.Single; break;
                 case CorElementType.ELEMENT_TYPE_R8:
                     typeCode = TypeCode.Double; break;
-                case CorElementType.ELEMENT_TYPE_STRING:
-                    typeCode = TypeCode.String; break;
```

Comment:
What drove this change?  This means that GetTypeCode will no longer return String for strings?

Reply:
`RuntimeTypeHandle.GetCorElementType` does not ever return `ELEMENT_TYPE_STRING`. `ELEMENT_TYPE_STRING` is only used as a shortcut in signatures. It is not used anywhere else in the system.

`TypeCode.String` is returned via the default case.

Reply:
Got it.